search/mongo_search.go

```diff
@@ -14,6 +14,8 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+var opInterruptedCode = 11601
```

Comment:
I found it using RoboMongo - it was consistently returned by the interrupted operations. **But** I did find the enumerations of the current mongodb error codes in mongo's source. 😃